### ISSUE TITLE
True JS expressions

### ISSUE THREAD

**Issue by mstijak** [codaxy/cxjs#285]
Expressions could be written in the following form:
```
<Widget value:expr={get=>get('x') + get('y)} />
```
Memoization is tricky to implement, but it might be possible to check bindings of the last run and invoke the function only when change is detected.

**Comment by yrodrigez** [comment]
Whats your idea? what `v` means? this looks better than a expression inside strings for sure...

**Comment by yrodrigez** [comment]
maybe something like this? 
`<Widget value:expr={
( storeVar1, storeVar2, storeFunc ) => storeFunc(storeVar1, storeVar2) } />`

**Comment by mstijak** [comment]
`v` means get value from the store.

**Comment by mstijak** [comment]
We also support [computables](https://docs.cxjs.io/concepts/data-binding#computables) which are similar to what you described.

**Comment by yrodrigez** [comment]
Then True JS expressions are already supported i guess?

**Comment by mstijak** [comment]
Yes, but this might be an alternative and somewhat shorter syntax.

**Comment by mstijak** [comment]
It would be even nicer to use proxies, but that wouldn't work in IE11.
Check the example here: https://github.com/kube/monolite

Alternatively, a babel plugin could transform expressions like  `d => d.x + d.y` into `v => v('x') + v('y')`, but that wouldn't work for TypeScript as it doesn't support such transformations. 
For TypeScript, one would have to rely on Proxy or use Babel for compilation (Babel 7 will support TS).
